LICENSE

```diff
@@ -1,2 +1,2 @@
 Copyright 2018 PDFTron Systems Inc. All rights reserved.
```

Comment:
Can we make this license 2024 Apryse? For example https://github.com/ApryseSDK/webviewer-ui/blob/10.12/LICENSE